### PR TITLE
Update deps to 0.14

### PR DESCRIPTION
**Pull request by orkunkl**

### CODE REVIEW

File: meta/test_generate.sh [hunk 1]
```diff
@@ -33,7 +33,5 @@ PROJECT_NAME="testgen-local"
 
     echo "Building wasm ..."
     cargo wasm
-    echo "Running integration tests ..."
```
Comment (orkunkl) [inline meta/test_generate.sh:36]: No integration tests in template anymore, no need for this line

File: src/msg.rs [hunk 2]
```diff
@@ -2,7 +2,7 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
-pub struct InitMsg {
+pub struct InstantiateMsg {
```
Comment (webmaster128) [inline src/msg.rs:5]: Nice. Could you rename `HandleMsg` -> `ExecuteMsg` as well?

**Review by webmaster128 — COMMENTED** [review]
Nice, thanks!

**Comment by ethanfrey** [comment]
Looks good.

You need to update to 1.51.0 here: https://github.com/CosmWasm/cosmwasm-template/blob/master/meta/appveyor.yml#L13

That and Simon's comment and good to move forward

**Comment by ethanfrey** [comment]
Looks good. We do need to update the schemas, but I realized no one has done that in over a year, so that can be a different PR. https://github.com/CosmWasm/cosmwasm-template/tree/master/schema

**Review by ethanfrey — APPROVED** [review]
Good stuff. Thanks

**Pull request merged**